README.md

```diff
@@ -37,124 +71,378 @@ To install the global command line interface (CLI), use npm global installation:
 npm install -g fqm-execution
 ```
 
-## Usage
+# Usage
+
+## Quickstart
+
+```typescript
+import { Calculator } from 'fqm-execution';
+
+// ...
 
-### API
+const { results } = await Calculator.calculate(measureBundle, patientBundles, options[, valueSetCache]);
 
-Import the necessary modules:
+// ...
+```
+
+## Requirements
+
+To calculate a FHIR-based eCQM, `fqm-execution` needs the following information:
+
+- A [FHIR Bundle](http://hl7.org/fhir/bundle.html) resource that contains:
+  - One [FHIR Measure](https://build.fhir.org/ig/HL7/cqf-measures/StructureDefinition-measure-cqfm.html) resource containing the metadata for the measure
+  - The [FHIR Library](https://build.fhir.org/ig/HL7/cqf-measures/StructureDefinition-library-cqfm.html) resource that is referenced from the above `Measure` resource's `.library` property, and any other FHIR `Library` resources that are dependencies of the main measure logic (e.g. `FHIRHelpers`)
+    - **NOTE**: These `Library` resources _must_ contain `base64`-encoded ELM JSON content directly on the resource. `fqm-execution` does not do any real-time translation of CQL to ELM
+  - Any [FHIR ValueSet](http://hl7.org/fhir/valueset.html) resources that are used in the measure logic\*
+- One or more FHIR `Bundle` resources that contain:
```

Comment:
Should we specify the instances in which we do not need patient bundles? Like dataRequirements and queryInfo? Or is this just for calculate

Reply:
The intent was the latter since it's under a sentence that says "to calculate a FHIR-based eCQM" (most common use case)

I'm gonna add an aside anyway though